js/tests/unit/tooltip.spec.js

```diff
@@ -387,6 +387,28 @@ describe('Tooltip', () => {
       tooltip.show()
     })
 
+    it('should show a tooltip whose class depends on the placement', done => {
```

Comment:
it seems not what you're testing here, because there are several other unit tests to handle that, to me your description should be: should replace placement right by end when placement specified

Reply:
and you should do the same for the `left` placement